noir-projects/aztec-nr/aztec/src/context/public_context.nr

```diff
@@ -133,18 +134,12 @@ impl PublicContext {
     }
     fn get_args_hash(mut self) -> Field {
         if !self.args_hash.is_some() {
-            let mut hasher = ArgsHasher::new();
+            let compute_args_hash = self.compute_args_hash;
```

Comment:
You can do `(self.compute_args_hash)()` inside the Option::some ;)

Reply:
I can't unfortunately ): noir does not handle it well

Reply:
It assumes that it will be a method not an stored fn

Reply:
Ahhh missed the extra parens!